tests/integrations/test_mlflow.py

```diff
@@ -10,20 +12,21 @@
 }
 
 
-def _evaluate(dataset, model, evaluator_config, request):
-    mlflow.end_run()
-    mlflow.start_run()
-    model_info = model.to_mlflow()
+def _evaluate(dataset, model, evaluator_config):
 
-    mlflow.evaluate(
-        model=model_info.model_uri,
-        model_type=mlflow_model_types[model.meta.model_type],
-        data=dataset.df,
-        targets=dataset.target,
-        evaluators="giskard",
-        evaluator_config=evaluator_config,
-    )
-    mlflow.end_run()
+    with TemporaryDirectory() as f:
+        mlflow.set_tracking_uri(Path("file://" + f))
```

Comment:
This is not a `Path`, it looks like a URI. Use `file:///path/to/the/dir`.

Reply:
If you want to pass a `Path` instance, just do

```python
mlflow.set_tracking_uri(Path(f))
```

Reply:
you're right, thanks!